provider/src/main/java/com/raygun/raygun4android/workers/RUMWorkerHelper.java

```diff
@@ -0,0 +1,22 @@
+package com.raygun.raygun4android.workers;
+
+import android.content.Context;
+import androidx.work.Data;
+import androidx.work.OneTimeWorkRequest;
+import androidx.work.WorkManager;
+import com.raygun.raygun4android.logging.RaygunLogger;
+
+public class RUMWorkerHelper extends RaygunWorkerHelper {
```

Comment:
I think there is no need to extend the `RaygunWorkerHelper` (same for the `CrashReportingWorkerHelper`) as they don't override or call the methods in `RaygunWorkerHelper`, as it is now.

Alternatively, `RaygunWorkerHelper` could be made abstract and have a generic `enqueue` function, since `enqueueRUM()` and `enqueueCrashReport()` have the same signature and share some code, although probably not worth it.